plugins/push/frontend/public/javascripts/countly.models.js

```diff
@@ -302,12 +418,148 @@
         },
     };
 
+    //NOTE: api object will reside temporarily in countlyPushNotification until countlyApi object is created;
```

Comment:
Separated the data access communication from the module service. Once, I can put in the countlyApi global object then I will move the interface inside it.